Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -1236,108 +1280,199 @@ suggestedPriceView push state =
-                          [ height $ V 24
-                          , width WRAP_CONTENT
-                          , color Color.darkCharcoal
-                          , text $ getString BOOKING_PREFERENCE
-                          , accessibility DISABLE
-                          , textSize FontSize.a_16
-                          , fontStyle $ FontStyle.regular LanguageStyle
-
-                          ],
-                          imageView
-                          [ width $ V 10
-                          , height $ V 10
-                          , margin (Margin 9 8 0 0)
-                          , accessibility DISABLE
-                          , imageWithFallback if state.data.showPreferences then fetchImage FF_COMMON_ASSET "ny_ic_chevron_up" else fetchImage FF_ASSET "ny_ic_chevron_down"
-                          ]
-                      ],
-                      linearLayout
-                        [ width MATCH_PARENT
-                        , height WRAP_CONTENT
-                        , margin $ MarginLeft 20
-                        , orientation VERTICAL
-                        ][ linearLayout
-                          [ width MATCH_PARENT
-                          , height WRAP_CONTENT
-                          , orientation VERTICAL
-                          , visibility if state.data.showPreferences then VISIBLE else GONE
-                          ][ showMenuButtonView push (getString AUTO_ASSIGN_DRIVER) ( fetchImage FF_ASSET "ny_ic_faster_lightning") true state,
-                            showMenuButtonView push (getString CHOOSE_BETWEEN_MULTIPLE_DRIVERS) (fetchImage FF_ASSET "ny_ic_info") false state]
-                      ]
+        ]
+    , linearLayout
+        [ width WRAP_CONTENT
+        , height MATCH_PARENT
+        , gravity CENTER_VERTICAL
+        ][ let fareEstimate = if state.data.rateCard.additionalFare == 0 then state.data.config.currency <> (show state.data.suggestedAmount) else  state.data.config.currency <> (show state.data.suggestedAmount) <> "-" <> state.data.config.currency <> (show $ (state.data.suggestedAmount + state.data.rateCard.additionalFare))
+           in
+            textView $
+            [ text $ fareEstimate
+            , width WRAP_CONTENT
+            , color Color.black900
+            , height WRAP_CONTENT
+            , fontStyle $ FontStyle.bold LanguageStyle
+            , accessibilityHint $ "Estimated Fare Is " <> fareEstimate
+            , onClick (\action -> if (getValueFromConfig "showRateCard") == "true" then push action else pure unit ) $ const ShowRateCard
+            ] <> FontStyle.body7 LanguageStyle
+         , imageView
+            [ imageWithFallback "ny_ic_info_blue,https://assets.juspay.in/nammayatri/images/common/ny_ic_info_blue.png"
+            , width $ V 40
+            , height $ V 40
+            , accessibility DISABLE
+            , visibility if (getValueFromConfig "showRateCard") == "true" then VISIBLE else GONE
```

Comment:
since you are using `(getValueFromConfig "showRateCard") == "true"` at multiple places it's better to use 
where clause